test/unit/kubernetes-deploy/runner_task_test.rb

```diff
@@ -3,52 +3,20 @@
 require 'kubernetes-deploy/runner_task'
 
 class RunnerTaskUnitTest < KubernetesDeploy::TestCase
-  def test_missing_namespace
```

Comment:
This and the last test in the file are the two unit tests I mentioned in my comment that require connecting to minikube. I moved them to the integration test file.